extensions/libarchive/ManipulateArchive.h

```diff
@@ -27,6 +27,9 @@
 #include "core/Processor.h"
 #include "core/ProcessSession.h"
 
+#include "FocusArchiveEntry.h"
+#include "UnfocusArchiveEntry.h"
```

Comment:
could these two includes be in the `cpp` file?

Reply:
done